<launch>
    <node name="px4flow" pkg="px4flow_node" type="px4flow_node" output="screen" respawn="true" clear_params="true">
        <rosparam file="$(find px4flow_node)/launch/px4flow_parameters.yaml"/>
    </node>

    <node name="true_velocity" pkg="px4flow_node" type="true_velocity_node.py" output="screen" respawn="true" clear_params="true">
        <rosparam file="$(find px4flow_node)/launch/px4flow_parameters.yaml"/>
    </node>
</launch>
